examples/jenkins/pipeline/jenkinstemplate.json

```diff
@@ -37,6 +37,10 @@
       "apiVersion": "v1",
       "metadata": {
         "name": "${JENKINS_SERVICE_NAME}",
+        "annotations": {
+          "service.alpha.openshift.io/dependencies": "[{\"name\": \"jenkins-jnlp\", \"namespace\": \"\", \"kind\": \"Service\"}]",
```

Comment:
Again, this is not a reference to a virtual resource.  This is a reference
to a concrete object.

On Mon, Jun 13, 2016 at 11:36 AM, Clayton Coleman <ccoleman@redhat.com>
wrote:

> Why in the world would I reference Scale here?
>
> On Mon, Jun 13, 2016 at 11:06 AM, David Eads <notifications@github.com>
> wrote:
>
>> In examples/jenkins/pipeline/jenkinstemplate.json
>> <https://github.com/openshift/origin/pull/9189#discussion_r66808349>:
>>
>> > @@ -37,6 +37,10 @@
>> >        "apiVersion": "v1",
>> >        "metadata": {
>> >          "name": "${JENKINS_SERVICE_NAME}",
>> > +        "annotations": {
>> > +          "service.alpha.openshift.io/dependencies": "[{\"name\": \"jenkins-jnlp\", \"namespace\": \"\", \"kind\": \"Service\"}]",
>>
>> Ugh, I'm really concerned about not using Kinds. Most clients are kind
>> focused when it comes to doing references.
>>
>> I think that Scale clearly demonstrates the failing of building
>> references based on an ambiguous attribute. This code seems to be looking
>> at a resource. We could put a resource into the kind field so it
>> serializes cleanly, but I worry about having an API that is already unable
>> to express certain kinds of references.
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/openshift/origin/pull/9189/files/1594d43a831b62264181448927589cd7e7124d01#r66808349>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe/ABG_p7HT_pTr8IYTm9qlVkBaQkLyeHUPks5qLXH_gaJpZM4IvZXy>
>> .
>>
>
>